### PR TITLE
Rewrite example and update readme.

### PR DESCRIPTION
**Pull request by pawaclawczyk**

### CODE REVIEW

File: examples/Application/SignUpHandler.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace examples\Clearcode\CommandBusConsole\Application;
+
+final class SignUpHandler
+{
+    public function handle(SignUp $command)
+    {
+        var_dump($command);
```
Comment (lzakrzewski) [inline examples/Application/SignUpHandler.php:9]: Leftover ?

Comment (pawaclawczyk) [inline examples/Application/SignUpHandler.php:9]: No it is not.

**Comment by clearcode** [comment]
:+1:

**Comment by krymen** [comment]
:+1:

**Pull request merged**